### DISCUSSION TITLE
Everything completed but '7 JSON files are missing'

### DISCUSSION THREAD

**Discussion by blingblongblah** [simulot/immich-go#571] [Q&A] [answered]
It looks everything is done, which is great, what an amazing project this is. I have received this message:

<img width="835" alt="Screenshot 2024-12-19 at 09 10 45" src="https://github.com/user-attachments/assets/70e1a6b9-d2c3-487c-a319-bc252fa6682f" />

Is this a cause for concern? Where are the logs? I thought they might be populated in the same folder but I can't see them there. 

TIA

**Comment by simulot** [comment, marked as answer]
Log path is written on the screen.
Usually, such a lower number of missing JSON is due to copied file, or edited images... probably not significant
You may force the integration of those file with the `-upload-when-missing-JSON` flag

**Reply by blingblongblah** [reply]
Thank you @simulot
